src/components/loadflow-result.js

```diff
@@ -144,22 +156,23 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
         [classes.cell, classes.fail, classes.succeed]
     );
 
-    const loadFlowConstraintscolumns = useMemo(() => {
+    const loadFlowCurrentViolationscolumns = useMemo(() => {
```

Comment:
loadFlowCurrentViolationsColumns instead of loadFlowCurrentViolationscolumns

Reply:
Done